common/src/main/java/com/sequenceiq/cloudbreak/logger/RestLoggerFilter.java

```diff
@@ -91,7 +95,17 @@ private static String logContent(byte[] content, String contentEncoding) {
         } catch (UnsupportedEncodingException e) {
             contentString = "We were not able to encode the content: " + e.getMessage();
         }
-        return AnonymizerUtil.anonymize(contentString);
+        return redact(contentString, hide);
```

Comment:
I think it would make sense to `redact` the content before checking the size, maybe the anonymisation can reduce the size.

Reply:
Anonymization is super expensive due to the complex regexps, and it does not reduce the size significantly, maybe it reduces with a few chars, but the limit is 30k already, so it would not make any difference.

I would like to keep it as it is.

Reply:
Thanks, makes sense as I didn't consider that instead of the too large messages we just simply return with a line.